### ISSUE TITLE
Error on Foxx service upgrade (ArangoError 1305: IO error: While appending to file: /data/engine-rocksdb/journals/003715.log: Out of memory)

### ISSUE THREAD

**Issue by henrycunh** [arangodb/arangodb#10999]
## My Environment

* __ArangoDB Version__:        3.6.0
* __Storage Engine__:             RocksDB
* __Deployment Mode__:        Cluster
* __Deployment Strategy__:   Kubernetes
* __Configuration__:               Running on 3 n2-highmem machines
* __Infrastructure__:               Google Cloud
* __Operating System__:       Alpine 3.10.3
* __Total RAM in your machine__:       13GB each, total of 49GB
* __Disks in use__:        10GB SSD each, total of 30GB
* __Used Package__:             Guide's .yaml deployment file

## Component, Query & Data
__Affected feature__:
Foxx

## Steps to reproduce

1. Download the service https://0x0.st/irFW.zip
2. Run `env NODE_TLS_REJECT_UNAUTHORIZED=0 foxx --verbose --database=<db> --server=https://<host>:<port> upgrade /server`
3. Verifiy the error
```
Unexpected ArangoDB error (Code: 1305):
ArangoError 1305: IO error: While appending to file: /data/engine-rocksdb/journals/003715.log: Out of memory
```

__Problem__:
On trying to upgrade a foxx microservice, or even trying to install one, 
Unexpected ArangoDB error (Code: 1305):
ArangoError 1305: IO error: While appending to file: /data/engine-rocksdb/journals/003715.log: Out of memory
__Expected result__:
Be able to upgrade my foxx service

**Comment by dothebart** [comment]
Hi, is this still reproducible with the latest arangodb?

**Comment by riteshapatel** [comment]
Yes. We have multiple Arango databases running on EKS and one of our database is showing this error. 

```
2020-11-09T16:57:06Z [1] WARNING [82af5] {statistics} could not commit stats to _statisticsRaw: IO error: While appending to file: /data/engine-rocksdb/journals/005020.log: Out of memory
```

Engine: RocksDB
Arango: 3.7.1
Deployment: ArangoDeployment

All databases deployed using `Arango Operator`. 

Any update on this issue? Is there a way to restart deployment? 

Thx

**Comment by dothebart** [comment]
Hi, 
Please upgrade to ArangoDB 3.7.3. This fixes a known issue (a memory leak) with 3.7.1 .

**Comment by riteshapatel** [comment]
@dothebart thank you for getting back. Will do.
